_posts/2021-08-24-pulp-community-roadmap-update.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Pulp Community Roadmap Update
+author: Melanie Corr
+tags:
+  - roadmap
+---
+
+Over the coming months, the Pulp team will focus on the following areas. As with all plans, this roadmap is subject to change. Many of the items in this plan factor into Katello and GalaxyNG roadmaps.
+Further updates will be announced on the Pulp blog and community. If you have any questions or feedback, feel free to write to post to our community channels!
+
+### OSTree plugin
+
+While Pulp 2 had an OSTree plugin, Pulp 3 was released without any OSTree functionality.
+The Pulp 2 OSTree plugin primarily managed Fedora Atomic Host content that has been EOL for several years.
```

Comment:
I'd leave it to @dkliban , however I suggest to mention examples when OSTree plugin is useful, why we decided to add it back, or do not mention Atomic Host and its EOL.
It's a bit confusing to me, I read it as OSTree plugin has nothing to manage, but hurray we are adding it :D

Reply:
My bad that was my suggestion on mentioning the going away stuff. @dkliban can perhaps provide some suggested wording to focus on current/future.